### PR TITLE
Cleanup .gitignore module

### PR DESCRIPTION
**Pull request by shulard**
I am targetting this branch, because this is a pedantic branch.

Related to: sonata-project/dev-kit#211
## Subject

Remove plateform specific ignore entries from .gitignore file.

### CODE REVIEW

**Comment by greg0ire** [comment]
Please amend your commit and force-push it to trigger the StyleCI build again, I don't know why it failed exactly…

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
+/Resources/doc/_build
+/vendor
+/node_modules
+/.sass-cache
 composer.lock
-vendor
-node_modules
-.sass-cache
+phpunit.xml
 .php_cs.cache
```
Comment (core23) [inline .gitignore]: Can you switch the position of `.php_cs.cache` like in the other projects.

Comment (shulard) [inline .gitignore]: Hello !

I've used the same order than on others projects here... Do you want that I move `node_modules`, `.sass-cache` at the end of the file ?

Comment (shulard) [inline .gitignore]: I'll use that order like on all the others projects :

```
/build
/Resources/doc/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
```

Do you want that I let the order folder / files or I put the global ignore at the beginning and the project specific after that ?

Example :

```
/build
/Resources/doc/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml

/node_modules
/.sass-cache
```

**Comment by greg0ire** [comment]
Please rebase, the StyleCI should be ok…

**Comment by shulard** [comment]
I've rebased my branch ;)

**Comment by greg0ire** [comment]
Great! Let's see if Travis passes, I have seen it fail for another, recent PR ( #748 ), for no apparent reason…

**Comment by greg0ire** [comment]
Oh it passes! Great!

**Comment by greg0ire** [comment]
@core23 please approve and merge

File: .gitignore [hunk 2]
```diff
@@ -1,12 +1,8 @@
...
-Resources/doc/_build/*
-nbproject
-coverage
-composer.lock
-vendor
-node_modules
-.sass-cache
+/.sass-cache
+/build
+/Resources/doc/_build
```
Comment (core23) [inline .gitignore]: Please sort this alphabetically

**Comment by shulard** [comment]
As mentioned in sonata-project/SonataPageBundle#68, I've removed the assets lines from .gitignore.

**Pull request merged**